media/css/firefox/new/fx-lifestyle-base.less

```diff
@@ -0,0 +1,283 @@
+    }
+
+    p {
+        .font-size(24px);
+        line-height: 38px;
+        margin-bottom: 50px;
+    }
+}
+
+#tabzilla:before {
+    background-color: @blue;
+}
+
+#newsletter-form {
+    display: none;
+}
+
+.fxfamilynav-header {
+    // Safari 5.1+, Chrome 10+
+    background-image: -webkit-linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0)),
+        -webkit-linear-gradient(@blue, @blue);
+    background-image: -o-linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0)),
+        -o-linear-gradient(@blue, @blue);
+    // the standard
+    background-image: linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0)),
+        linear-gradient(@blue, @blue);
+}
+
+.content-section {
+    color: #484848;
+    padding: 50px 0;
+
+    @media only screen and (max-width: @breakTablet) {
+        padding: 50px 0;
+        text-align: center;
+    }
+}
+
+.content {
+    .clearfix();
+    background-repeat: no-repeat;
+    margin: 0 auto;
+    width: @widthDesktop - 20px;
+
+    a {
+        color: @blue;
+        font-weight: bold;
+    }
+
+    @media only screen and (max-width: @breakDesktop) {
+        width: @widthTablet - 20px;
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        background-position: bottom 20px center;
+        width: @widthMobileLandscape - 20px;
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        padding-left: 10px;
+        padding-right: 10px;
+        width: @widthMobile - 20px;
+    }
+}
+
+h2 {
+    .font-size(52px);
+    line-height: 1.3;
+    margin-bottom: 25px;
+    text-shadow: none;
+
+    @media only screen and (max-width: @breakDesktop) {
+        .font-size(46px);
+        margin-bottom: 34px;
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        .font-size(38px);
+        margin-bottom: 30px;
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        .font-size(28px);
+    }
+}
+
+h3 {
+    .font-size(29px);
+    line-height: 1.4;
+    margin-bottom: 40px;
+    text-shadow: none;
+
+    @media only screen and (max-width: @breakDesktop) {
+        .font-size(30px);
+        line-height: 1.3;
+        margin-bottom: 25px;
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        .font-size(24px);
+    }
+}
+
+.copy {
+    width: 600px;
+    padding: 0 30px;
+
+    p {
+        .font-size(18px);
+        .open-sans-light();
+        line-height: 40px;
+    }
+
+    @media only screen and (max-width: @breakDesktop) {
+        width: 420px;
+
+        p {
+            .font-size(16px);
+            line-height: 34px;
+        }
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        width: auto;
+        padding: 0 0;
+
+        p {
+            .font-size(14px);
+            line-height: 30px;
+        }
+    }
+}
+
+.fx-privacy-link {
+    a:active,
+    a:focus,
+    a:hover,
+    a:link,
+    a:visited {
+        color: #d9d9d9;
```

Comment:
This color is too light for the download button on the white background. Should make a different style for that case.